hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -653,14 +653,6 @@ ContainerCommandResponseProto handleDeleteChunk(
       return malformedRequest(request);
     }
 
-    // The container can become unhealthy after the lock is released.
-    // The operation will likely fail/timeout in that happens.
-    try {
-      checkContainerIsHealthy(kvContainer);
-    } catch (StorageContainerException sce) {
-      return ContainerUtils.logAndReturnError(LOG, sce, request);
-    }
```

Comment:
Where is this check available in ``handleDeleteChunk`` path? I tried checking callers of the method, couldn't find where this is checked before. Can you point me to that

Reply:
Sorry, I didn't get the question? Do you mean where the function is called before?

Reply:
Yes

Reply:
I didn't find the usage of DeleteChunk either. 
I think the method of `deleteChunk` in `ChunkManager` is designed for `FilePerChunkStrategy` first that it will only delete one chunk file, in `FilePerBlockStrategy` the method should throw the exception of unsupported request.